### DISCUSSION TITLE
Unresponsive input after page transition

### DISCUSSION THREAD

**Discussion by PietM** [swup/swup#963] [Help] [answered]
### Describe your problem 🧐

Hi everyone, 
I've encountered an issue with input textareas / fields for embedded forms re: Mailchimp. I've noticed that input textareas lose functionality (can't input anything) after transitioning to the page with the embedded form / field. Even after removing any java, the form doesn't respond once engaged. It's like the keyboard has been unplugged. 

### What is your question? 🙋‍♀️

How do I restore the basic functionality of the input textarea / form once I transition to the page with the embedded form / field?

### URL to minimal reproduction 🔗

https://arp.work/FINALCC/contact/

### Checked all these? 📚

- [X] I have read and searched the [official docs](https://swup.js.org)
- [X] I have checked [discussions](https://github.com/orgs/swup/discussions) and [existing issues](https://github.com/issues?q=is:issue+org:swup) for answers to my question
- [X] I agree to follow this project's [Code of Conduct](https://github.com/swup/.github/blob/master/CODE_OF_CONDUCT.md)
- [X] I have provided all necessary information to the best of my knowledge

**Comment by daun** [comment, marked as answer]
It's only broken when navigating to Contact from the homepage. From other pages (.e.g. About), it seems to be working normally. I also noticed that on the homepage, normal keyboard events get swallowed by some event listener on the page. Cmd+R doesn't reload the page, Cmd+alt+I doesn't open devtools, etc. Looks like that's the culprit.

**Reply by PietM** [reply]
Thanks for this, @daun! 
You're right. 
The issue has nothing to do with pagination or swup, but a control on "keydown" events. I can't thank you enough for swup and for generously taking the time to look into my issue. 
Really. Thank you!

**Reply by daun** [reply]
Great, glad you could solve it!
